src/gui/src/scriptWidget.cpp

```diff
@@ -332,6 +332,8 @@ void ScriptWidget::pause()
   pauser_->setEnabled(true);
   paused_ = true;
 
+  input_->setReadOnly(true);
```

Comment:
There are tcl commands that only affect GUI state and could be used during pause (e.g. zoom, highlight).  I don't think we want to exclude all commands

Reply:
When you type something in at pause it effectively breaks out the current execution loop (if you press enter). I'm not sure what happens when the pause is initiated by a tool, but I think it would be undesirable, since the interpreter is still running the tool code.
```
for {set x 0} {$x < 10} {incr x} {
puts "Here: $x"
gui::pause
}
```
If I type gui::fit after the first pause I get:
```
Here: 0
>>> gui::fit
```
Instead of:
```
Here: 0
>>> gui::fit
Here: 1
```
Which would be the expected behavior.

Reply:
Well, nevermind... It did actually continue, but the buttons had all reset (so it appeared to have stopped the loop).  I can revert that change out, but at the moment other parts of the scriptWidget do not behave as expected, so those will need to get corrected.

Reply:
Ok let's leave it as you have it and consider this a separate issue.  I can imagine other problems with reentering tcl during a command.